types/api/v1beta1/event_data.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+import "google/protobuf/wrappers.proto";
+
+message EventValue {
+    oneof value {
+        google.protobuf.Int32Value int32 = 1; // intT
+        google.protobuf.UInt32Value uint32 = 2; // uintT
+        google.protobuf.Int64Value int64 = 3; // longT
+        google.protobuf.UInt64Value uint64 = 4; // ulongT
+        google.protobuf.UInt64Value off = 5; // offT
+        google.protobuf.UInt32Value mode = 6; // modeT
+        google.protobuf.UInt32Value dev = 7; // devT
+        google.protobuf.UInt64Value size = 8; // sizeT
+        google.protobuf.UInt64Value pointer = 9; // pointerT
```

Comment:
do we need to repeat those types?

Reply:
oh, true, we have the name in the map, so we don't have to repeat the types here.